packages/core/src/appformer/AppFormer.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from "react";
+import { Element } from "../core";
+import { Screen } from "./Screen";
+import { Perspective } from "./Perspective";
+
+export class AppFormer {
+  public init(container: HTMLElement, callback: () => void): AppFormer {
+    return this;
+  }
+
+  public registerScreen(screen: Screen): void {
+    //
```

Comment:
Sure! Good suggestion.

Reply:
What do you thing about logging someting here? Maybe it would be good to have some logger enabled. We could the use it in `AppFormer class in empty method bodies e.g.

`log.info("Registering Screen" + screen  )`? However I am not familiar with any logger package that could be used, so I can't suggest anything. WDYT? Woudl something like that be useful for us?